### PR TITLE
Update to go 1.16

### PR DESCRIPTION
**Pull request by chriskurzeja**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-FROM golang:1.15
+FROM golang:1.16-alpine3.13
 
-RUN apt-get update && apt-get -y install zip && rm -rf /var/lib/apt/lists/*
+RUN apk add --no-cache --update git openssh-client make bash lftp zip
```
Comment (mikemoate) [inline Dockerfile]: @randomjunk this comes from https://github.com/Countingup/docker-node, I assume you figured out what Sempahore "needs" here?

Comment (chriskurzeja) [reply Dockerfile]: This was missing `coreutils` based on the semaphore docs

Comment (randomjunk) [reply Dockerfile]: there's a very specific list:
https://docs.semaphoreci.com/ci-cd-environment/custom-ci-cd-environment-with-docker/#building-custom-docker-images

& `make` which we require

File: README.md [hunk 2]
```diff
@@ -2,8 +2,13 @@
...
+Minimal golang:1.16-alpine3.13 base image with a few tools useful in CI jobs.
 
 Includes:
 
 - zip
+- git
+- ssh
+- make
+- bash
+- lftp
```
Comment (mikemoate) [inline README.md:14]: (nitpick) Mention coreutils here.

Comment (chriskurzeja) [reply README.md:14]: Legit. Wilco

Comment (chriskurzeja) [reply README.md:14]:
```suggestion
- zip
- git
- ssh
- make
- bash
- lftp
- coreutils
```

**Review by randomjunk — APPROVED** [review]

**Pull request merged**